### PR TITLE
Update lambda config for failure

### PR DESCRIPTION
**Pull request by jzbahrai**
# Summary | Résumé
Update lambda config: https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/lambda_function_event_invoke_config .

Goal is to reduce invocation on failure.

### CODE REVIEW

File: aws/heartbeat/lambda.tf [hunk 1]
```diff
@@ -15,6 +15,12 @@ module "heartbeat" {
   }
 }
 
+resource "aws_lambda_function_event_invoke_config" "heartbeat_" {
+  function_name                = module.heartbeat.function_name
```
Comment (jimleroyer) [inline aws/heartbeat/lambda.tf:19]: Just tagging @patheard to know if that will work via the module this way. The plan seems to work and define some changes.

**Review by jimleroyer — COMMENTED** [review]
LGTM, just want Pat to confirm!

File: aws/heartbeat/lambda.tf [hunk 2]
```diff
@@ -15,6 +15,12 @@ module "heartbeat" {
   }
 }
 
+resource "aws_lambda_function_event_invoke_config" "heartbeat_" {
```
Comment (patheard) [inline aws/heartbeat/lambda.tf:18]: nit: just around naming, I'd either remove the trailing `_` from the `heartbeat_` resource name or call it something like `heartbeat_invoke_config`.  Totally a preference thing though.

**Review by patheard — APPROVED** [review]
Look good!  Left a minor non-blocking comment but I'm curious to see if this works as advertised.

File: aws/heartbeat/lambda.tf (hunk 2 above)
Comment (jzbahrai) [reply aws/heartbeat/lambda.tf]: this was an error on my part 🤦🏽 I meant to write a whole word.

**Pull request merged**